ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -523,282 +316,193 @@ private void loadProperties() {
+        String proxyValue = cmd.getOptionValue(PROXY.getOpt());
+        if (Guard.isStringNullOrEmpty(proxyValue)) {
+            proxyValue = System.getenv("http_proxy");
+        }
+        addToLog("Proxy: " + proxyValue);
+        return proxyValue;
     }
 
-    public String getSecretKey() {
-        return this.secretKey;
+    // conveninece getters for private options
+    public String getVersion() {
+        return this.version;
     }
-
-    void setSecretKey(final String secretKey) {
-        this.secretKey = secretKey;
+    public String getBuildDate() {
+        return this.buildDate;
     }
-
     public String getCommand() {
-        return this.command;
-    }
-
-    void setCommand(final String command) {
-        this.command = command;
-    }
-
-    void setObjectName(final String objectName) {
-        this.objectName = objectName;
-    }
-
-    public String getObjectName() {
-        return this.objectName;
+        return cmd.getOptionValue(COMMAND.getOpt());
     }
-
-    void setProxy(final String proxy) {
-        this.proxy = proxy;
-    }
-
-    public String getProxy() {
-        return this.proxy;
-    }
-
-    void setForce(final boolean force) {
-        this.force = force;
+    public String getHelp() {
+        return cmd.getOptionValue(COMMAND_HELP.getLongOpt());
     }
-
-    public boolean isForce() {
-        return this.force;
-    }
-
-    void setRetries(final int retries) {
-        this.retries = retries;
+    public ViewType getOutputFormat() {
+        return ViewType.valueOf(this.getOptionValueWithDefault(VIEW_TYPE.getLongOpt(), ViewType.CLI).toString().toUpperCase());
     }
 
+    public boolean isCertificateVerification() { return !this.optionExists(INSECURE.getLongOpt()); }
+    public boolean isHttps() {  return !this.optionExists(HTTP.getLongOpt()); }
+    public boolean isHelp() { return this.help; }
+    void setHelp(final boolean help) { this.help = help; }
+    public Level getConsoleLogLevel() { return this.consoleLogLevel; }
+    void setConsoleLogLevel(final Level console) {this.consoleLogLevel = console; }
+    public Level getFileLogLevel() { return this.fileLogLevel; }
+    void setFileLogLevel(final Level file) {this.fileLogLevel = file; }
     public int getRetries() {
-        return this.retries;
-    }
-
-    public String getPrefix() {
-        return this.prefix;
-    }
-
-    void setPrefix(final String prefix) {
-        this.prefix = prefix;
-    }
-
-    public boolean isChecksum() {
-        return this.checksum;
-    }
-
-    void setChecksum(final boolean checksum) {
-        this.checksum = checksum;
+        final String retryString = this.getOptionValueWithDefault(RETRIES.getOpt(), DEFAULT_RETRIES).toString();
+        try {
+            return Integer.parseInt(retryString);
+        } catch (final NumberFormatException e) {
+            System.err.printf("Error: Argument (%s) to '-r' was not a number\n", retryString);
```

Comment:
This should follow the normal exception handling path and let the caller handle the printing of the exception.